crates/ursa-network/src/discovery.rs

```diff
@@ -91,16 +107,9 @@ impl DiscoveryBehaviour {
             peers.insert(peer_id);
         }
 
-        // boostrap
-        if let Err(error) = kademlia.bootstrap() {
-            warn!("Failed to bootstrap with Kademlia: {}", error);
-        }
-
-        let mdns = if config.mdns {
-            Some(Mdns::new(Default::default()).expect("mDNS start"))
-        } else {
-            None
-        };
+        let mdns = config
```

Comment:
Revert

Reply:
Sure thing, although was making it consistent with the usage of `.then_some()` a few lines up for the next_bootstrap

Reply:
Can you make that one consistent with this?

Reply:
https://github.com/fleek-network/ursa/pull/162/commits/1fca91c9621f84a47e18395fb8b17b65bc97b4f2